packages/mutation-testing-elements/src/components/mutation-test-report-popup/mutation-test-report-popup.scss

```diff
@@ -0,0 +1,35 @@
+:host {
+    position: relative;
+    display: inline-block;
+    user-select: none;
+}
+
+/* The actual popup */
+.popover {
+    width: 200px;
+    position: absolute;
+    z-index: 1;
+    top: 125%;
+    left: 50%;
+    margin-left: -100px;
+    // visibility: hidden;
```

Comment:
This should be enabled! Right now, a popup with opacity 1 can be blocking a mutant!